scss/adminLessonNav.module.scss

```diff
@@ -4,30 +4,33 @@
 $dark: hsl(0, 0%, 33.7%);
 $light: hsl(0, 0%, 98.4%);
 $light_secondary: hsl(0, 0%, 92.9%); // Border
+$dark_primary: color.adjust(variables.$primary, $lightness: -20%);
+$light_primary: color.adjust(variables.$primary, $lightness: 28%);
 
 .lessons_tabsNav {
   font-family: 'PT Mono';
-  width: 100%;
-  padding: 18px 23px;
-
+  max-width: fit-content;
+  padding: 10px 7px;
   background: white;
-  border: 1px solid $light_secondary;
-  box-shadow: 0px 4px 25px hsla(0, 0%, 0%, 0.1);
-  border-radius: 10px;
+  box-shadow: 0px 4px 25px hsla(0deg, 0%, 0%, 0.1);
+  border-radius: 4px;
   align-items: center;
   display: flex;
   column-gap: 20px;
+  overflow-x: auto;
```

Comment:
Used so a scrollbar would appear when the element's width is greater than the viewport size.